### DISCUSSION TITLE
Insert a CUT into a model based on file.FOLD?

### DISCUSSION THREAD

**Discussion by chatyan** [amandaghassaei/OrigamiSimulator#53] [Q&A]
This is a wonderful tool!!! Thank you so much. 
Can I insert a cut into a model based on file.FOLD.

I tried to change the edges_assignment to "B", but the model is still connected. 
Do you have any advice for me to cut in a model by modifying the file based on FOLD format?

Thank you.

**Comment by edemaine** [comment]
In the FOLD format, you need to actually separate (duplicate) the relevant mesh vertices and edges in order to cut them apart.

The [FOLD JavaScript library](https://github.com/edemaine/fold#fold-javascript-library) offers [`FOLD.filter.cutEdges`](https://github.com/edemaine/fold/blob/main/doc/api.md#foldfilter) to do the necessary mesh surgery for cutting edges. I don't think it yet works in all cases, but it often does.
